provider/k8s/k8s.go

```diff
@@ -263,8 +274,28 @@ func (p *Provider) initializeTemplates() error {
 	}
 
 	if p.CertManager {
-		if err := p.applySystemTemplate("cert-manager", nil); err != nil {
-			return errors.WithStack(err)
+		d, _ := p.Cluster.AppsV1().Deployments("cert-manager").Get("cert-manager", am.GetOptions{})
+		if d == nil {
+			return p.applySystemTemplate("cert-manager", nil)
+		}
+
+		if d.Spec.Template.Labels["app.kubernetes.io/version"] != "v1.9.0" {
```

Comment:
We should probably use an env variable for the cert-manager version so we don't need to change it every time we update its version. For now I think it's ok though, we can do it on the next iteration.